src/app/loadout-drawer/LoadoutDrawerContents.tsx

```diff
@@ -343,8 +345,8 @@ export function fillLoadoutFromEquipped(
       item.equipped &&
       itemCanBeInLoadout(item) &&
       (category
-        ? category === 'subclass'
-          ? item.bucket.hash === BucketHashes.Subclass
+        ? category === 'General'
+          ? item.bucket.hash !== BucketHashes.Subclass && item.bucket.sort === category
```

Comment:
I couldn't find a place where we were passing in `subclass` so I just deleted it. Now, when we are dealing with the general category we will fill everything but subclasses.

Reply:
I think that's just from before I switched to `setLoadoutSubclassFromEquipped`